src/service/room.service.ts

```diff
@@ -35,15 +35,33 @@ export class RoomService {
         headers: HEADER,
       });
       if (response.ok) {
-        return Result.success(ROOM_CREATE_SUCCESS);
+        return await Result.success(ROOM_CREATE_SUCCESS);
```

Comment:
`success`는 async 함수가 아니라 await를 붙이지 않아도 괜찮습니다. 
```suggestion
        return Result.success(ROOM_CREATE_SUCCESS);
```